### PR TITLE
LPS-180649 Add highlighting integration tests

### PR DESCRIPTION
**Pull request by joshuacords**
https://issues.liferay.com/browse/LPS-180649

**Author:** @joshuacords

### CODE REVIEW

**Comment by joshuacords** [comment]
ci:test:relevant

**Comment by joshuacords** [comment]
ci:test:stop

**Comment by joshuacords** [comment]
ci:test:stop

**Comment by joshuacords** [comment]
ci:stop

**Comment by joshuacords** [comment]
ci:stop

**Comment by joshuacords** [comment]
ci:stop

**Comment by joshuacords** [comment]
ci:test:sf

**Comment by joshuacords** [comment]
ci:test:relevant

**Comment by joshuacords** [comment]
ci:test:search

File: modules/apps/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/BaseSearchRequestBuilderHighlightTestCase.java [hunk 1]
```diff
@@ -73,18 +74,23 @@ public void testSearchWithHighlight() throws Exception {
...
-		).fragmentSize(
-			20
-		).postTags(
-			"[/H]"
-		).preTags(
-			"[H]"
-		).build();
+		HighlightBuilder highlightBuilder = _highlightBuilderFactory.builder();
+
+		List<String> fieldNames = Arrays.asList(getFieldNames());
```
Comment (gustavolimav) [inline modules/apps/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/BaseSearchRequestBuilderHighlightTestCase.java:79]: Maybe Bchan would like to change the order 

            List<String> fieldNames = Arrays.asList(getFieldNames());

            HighlightBuilder highlightBuilder = _highlightBuilderFactory.builder();

**Review by gustavolimav — CHANGES_REQUESTED** [review]
You rock @joshuacords! Love the changes, it's really clean and better than before. I will send this PR as a good example of refactoring to Felipe. The commits are in good order too! good job. take a look at my comment, it's just a small change thinking about what Bchan would like to change.

File: modules/apps/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/BaseSearchRequestBuilderHighlightTestCase.java (hunk 1 above)
Comment (joshuacords) [reply modules/apps/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/BaseSearchRequestBuilderHighlightTestCase.java:76]: I agree, thanks for catching this and thanks for the kind words! :blush:

**Comment by joshuacords** [comment]
ci:test:sf

**Comment by joshuacords** [comment]
ci:test:relevant

**Comment by joshuacords** [comment]
ci:test:sf

**Comment by joshuacords** [comment]
ci:test:sf

**Pull request closed without merging**